modules/purge_kestra/main.tf

```diff
@@ -11,7 +11,6 @@ module "kestra_purge_db" {
   postgres_database        = "kestra"
   postgres_password_secret = var.postgres_password_secret
   postgres_host            = var.postgres_host
-  show_result_in_logs      = false
```

Comment:
Just a comment, FYI there will be a Purge.Logs task in upcoming 0.18 version 😉 probably update here when release is shipped